.github/workflows/build.yml

```diff
@@ -241,6 +241,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Set up QEMU
```

Comment:
Note: There's a PR open to update the docker-build action and include this step there.. once that is merged, I'll remove this step and bump the docker build action